packages/react-magma-dom/src/components/DatePicker/CalendarMonth.tsx

```diff
@@ -196,8 +195,8 @@ export const CalendarMonth: React.FunctionComponent<CalendarMonthProps> = (
               </tbody>
             </Table>
             <Tooltip
-              content={'Keyboard instructions'}
-              tooltipStyle={{ position: 'fixed' }}
+              content={'Keyboard instructions\nctrl+shift+?'}
```

Comment:
@orion-cengage Do you have a preference between lowercase or uppercase for the tooltip?

Reply:
I used lowercase because saw them in the issue:

https://github.com/cengage/react-magma/issues/1396

![image](https://github.com/user-attachments/assets/2c05b0b6-7d4f-48d4-b51d-c139e0566b8c)

Reply:
Lower case is great